README.md

```diff
@@ -48,7 +48,8 @@ This tap:
     ```json
```

Comment:
Add the description for the request_timeout in point 3

Reply:
Added